.pre-commit-config.yaml

```diff
@@ -23,7 +23,8 @@ repos:
           (?x)^(
               .*\.lock |
               .*\.json |
-              .*\.ipynb
+              .*\.ipynb |
+              .*\.cppipe
```

Comment:
What does adding cppipe do when in the pre-commit config? Will pre-commit try to do anything with it?

Reply:
This line was added to help avoid spellchecks for the `cppipe` file (which otherwise was triggered by data found within that file). Spellcheck gets upset with data files generally, so this is following the convention along with the others above this line.